sig-service-catalog/charter.md

```diff
@@ -0,0 +1,165 @@
+  - Responsible for project administration activities within the SIG and are
+    non-technical in nature, such as organizing the weekly meetings.
+  - A chair does not have more rights, or votes, than a maintainer.
+  - Responsible for reporting the SIG’s status to the appropriate Kubernetes
+    leadership teams.
+  - All decisions amongst chairs are made using lazy consensus with a fallback
+    to a 2/3 majority vote (lazy two-thirds resolution).
+    This process is used for all decisions, such as changing chairs/maintainers
+    or modifying this charter.
+  - Chairs may nominate a new chair at any time, to be accepted via lazy
+    two-thirds resolution amongst the chairs.
+  - Chairs may decide to step down at any time.
+  - Chairs must remain active in the role and may be removed from the position
+    via lazy two-thirds resolution amongst the chairs, if they are unresponsive
+    for > 3 months or are not proactively working with other chairs to fulfill
+    responsibilities.
+  - There is no set number of Chairs.
+
+- Emeritus Chairs ([Inspired by the Helm
+  Project](http://technosophos.com/2018/01/11/introducing-helm-emeritus-core-maintainers.html))
+  - A chair who steps down may choose to take an Emeritus Chair title. This
+    confers honor on the recipient and allows them to remain associated with the
+    SIG in acknowledgement of their significant contributions.
+  - Those who attain this title are no longer expected to attend the weekly
+    meetings, share in the issue queue triage rotation, vote on policy or
+    architectural issues, or review pull requests.
+  - They are listed in our documentation as Emeritus Chairs, and we will
+    continue to invite them to participate in related events, such as KubeCon.
+
+- Security Contacts
+  - Are a contact point for the Product Security Team to reach out to for
+    triaging and handling of incoming issues.
+  - Must be a maintainer.
+  - Must accept and adhere to the Kubernetes [Embargo
+    Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#embargo-policy).
+  - Defined in
+    [SECURITY_CONTACTS](https://github.com/kubernetes-incubator/service-catalog/blob/master/SECURITY_CONTACTS)
+    file.
+
+## Organizational management
+
+- SIG meets every week on Zoom at 1 PM PST on Mondays
+    - Agenda
+      [here](https://docs.google.com/document/d/17xlpkoEbPR5M6P5VDzNx17q6-IPFxKyebEekCGYiIKM/edit#).
+    - Anyone is free to add new agenda items to the doc.
+    - Recordings of the calls are made available [here](https://goo.gl/ZmLNX9).
+- SIG members explicitly representing the group at conferences (SIG progress
+  reports, deep dives, etc.) should make their slides available for perusal and
+  feedback at least 2 week in advance.
+- [Working
+  groups](https://docs.google.com/document/d/1fghxARW-doHrNmBYCijhODsGoIFawpeU4X0end-XnUI/edit#)
```

Comment:
Just to keep our doc-sprawl to a minimum, can we move this gdoc into a wiki page in our repo? Then its still easy to update but doesn't require a PR to do so.

Reply:
Yeah I was just linking to whatever doc was available but it would be better to have that in one place. I'll copy it into our wiki and update the charter to link to that.